Jenkinsfile.groovy

```diff
@@ -46,12 +46,12 @@ pipeline {
             $class: 'GitSCM',
             branches: scm.branches,
             extensions: scm.extensions + [[$class: "CleanCheckout"]] + [[$class: "LocalBranch", localBranch: "**"]] + [[$class: 'CloneOption', noTags: false, reference: '', shallow: false]],
-            userRemoteConfigs: [[credentialsId: "github-coveobot", url: "https://github.com/coveo/platform-client.git"]]
+            userRemoteConfigs: [[credentialsId: "github-app-dev", url: "https://github.com/coveo/platform-client.git"]]
           ])
 
           sh "git config --global user.email \"jenkins@coveo.com\""
           sh "git config --global user.name \"Jenkins CI\""
-          sh "git remote set-url origin \"https://${env.GIT_USR}:${env.GH_TOKEN}@github.com/coveo/platform-client.git\""
+          sh "git remote set-url origin \"https://${env.GIT_USR}:${env.GIT_PSW}@github.com/coveo/platform-client.git\""
```

Comment:
I don't think this is useful anymore because Git operations require the use of either a personal, OAuth, or installation token. (or, although this is not preferred te use of a SSH token, such as a deploy key)

Thus, you should be able to target the repo itself without the user@password shenanigans.

Reply:
@toofff proposed himself to investigate how we could improve that, in the mean time I'm pretty sure it would fix the issues we're facing

Reply:
I don't mind merging it (so I'll give my approval on behalf of DX in that regard), but I personally do not think that this will fix the issue.